File: ReviewsJuegos/AgregarPlataforma.vb
Imports System.Data.SqlClient

Public Class AgregarPlataforma
    Private Sub BtnAgregar_Click(sender As Object, e As EventArgs) Handles BtnAgregar.Click
        ' Crear parámetros para el procedimiento almacenado
        Dim IdUsuario As Integer = 1
        Dim parametros(1) As SqlParameter
        parametros(0) = New SqlParameter("@NombrePlataforma", TxtAgPlata.Text)
        parametros(1) = New SqlParameter("@IdUsuarioCrea ", IdUsuario)




        ' Ejecutar el procedimiento almacenado
        ConexionSQL.ExecuteStoredProcedure("SP_AgregarPlataforma", parametros)
        MsgBox("Registro exitoso")
    End Sub

    Private Sub BtnCerrar_Click(sender As Object, e As EventArgs) Handles BtnCerrar.Click
        Me.Close()

    End Sub
End Class

File: ReviewsJuegos/AgregarPlataforma.cs
using System;
using System.Data.SqlClient;
using Microsoft.VisualBasic;

namespace ReviewsJuegos
{

    public partial class AgregarPlataforma
    {
        public AgregarPlataforma()
        {
            InitializeComponent();
        }
        private void BtnAgregar_Click(object sender, EventArgs e)
        {
            // Crear parámetros para el procedimiento almacenado
            int IdUsuario = 1;
            var parametros = new SqlParameter[2];
            parametros[0] = new SqlParameter("@NombrePlataforma", TxtAgPlata.Text);
            parametros[1] = new SqlParameter("@IdUsuarioCrea ", IdUsuario);




            // Ejecutar el procedimiento almacenado
            ConexionSQL.ExecuteStoredProcedure("SP_AgregarPlataforma", parametros);
            Interaction.MsgBox("Registro exitoso");
        }

        private void BtnCerrar_Click(object sender, EventArgs e)
        {
            Close();

        }
    }
}